src/utils/__mocks__/node-worker-contents.js

```diff
@@ -0,0 +1,24 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+// @flow
+// $FlowExpectError Flow doesn't know about this module
+const { parentPort, workerData } = require('worker_threads');
+const fs = require('fs');
+const vm = require('vm');
+
+if (!workerData) {
+  throw new Error(`Please pass a file name using the 'workerData' property.`);
+}
+
+const scriptContent = fs.readFileSync(workerData, 'utf8');
+
+const sandbox = {
+  importScripts: function() {},
```

Comment:
We don't use it in the worker so I didn't implement it. But it needs to be here for the emscripten blob to think it's running in a Worker.

Reply:
Is it called? Perhaps we can throw an error in it.

Reply:
good idea, so that if we call it in a worker in some future code, it's more obvious that we need to implement it.